tools/lkl/lib/virtio_net.c

```diff
@@ -124,6 +128,22 @@ void poll_thread(void *arg)
 	}
 }
 
+struct virtio_net_dev *registered_devs[MAX_NET_DEVS];
+static int registered_dev_idx = 0;
```

Comment:
There is another counter in lkl_netdev_add, I think we can remove it from now and use this one.